src/test/resources/scenario/FabCar.feature

```diff
@@ -10,24 +10,44 @@ Feature: Configure Fabric using SDK and submit/evaluate using a network Gateway
 		And I deploy node chaincode named fabcar at version 1.0.0 for all organizations on channel mychannel with endorsement policy 1AdminOr2Other and arguments ["initLedger"]
 
  	Scenario: Using a Gateway I can submit and evaluate transactions on instantiated node chaincode
-		Given I have a gateway as user User1 using the tls connection profile
-		And I connect the gateway
-		And I use the mychannel network
-		And I use the fabcar contract
-		When I prepare a createCar transaction
-	 	And I submit the transaction with arguments ["CAR10", "Trabant", "601 Estate", "brown", "Simon"]
-		And I prepare a queryCar transaction
-	 	And I evaluate the transaction with arguments ["CAR10"]
-		Then the response should be JSON matching
-		    """
-		    {
-		    	"color": "brown",
-		    	"docType": "car",
-		    	"make": "Trabant",
-		    	"model": "601 Estate",
-		    	"owner": "Simon"
-		    }
-		    """
+        Given I have a gateway as user User1 using the tls connection profile
+        And I connect the gateway
+        And I use the mychannel network
+        And I use the fabcar contract
+        When I prepare a createCar transaction
+        And I submit the transaction with arguments ["CAR10", "Trabant", "601 Estate", "brown", "Simon"]
+        And I prepare a queryCar transaction
+        And I evaluate the transaction with arguments ["CAR10"]
+        Then the response should be JSON matching
+            """
+            {
+                "color": "brown",
+                "docType": "car",
+                "make": "Trabant",
+                "model": "601 Estate",
+                "owner": "Simon"
+            }
+            """
+
+    Scenario: Using a Gateway with an X509Identity I can submit and evaluate transactions on instantiated node chaincode
+        Given I have a gateway with identity User1 using the tls connection profile
+        And I connect the gateway
+        And I use the mychannel network
+        And I use the fabcar contract
+        When I prepare a createCar transaction
+        And I submit the transaction with arguments ["CAR10", "Trabant", "601 Estate", "brown", "Simon"]
```

Comment:
A different ID than `CAR10` should be used. That ID is used by the scenario test above so it is possible one test may not be working correctly but appear that it is since it is seeing state set by the other test. Alternatively, you could use the same ID but with different information so we know we're querying the correct state.